packages/ui/core-components/src/lib/unsorted/viz/core/ECharts.svelte

```diff
@@ -183,7 +187,6 @@
 	.chart {
 		-moz-user-select: none;
 		-webkit-user-select: none;
-		-ms-user-select: none;
```

Comment:
Why was this removed?

Reply:
that i'm not sure - wasn't intentional. just added it back